src/Stan/Inspection/AntiPattern.hs

```diff
@@ -388,3 +391,17 @@ stan0215 = mkAntiPatternInspection (Id "STAN-0215") "Slashes in paths" (FindAst
     pathLit :: PatternAst
     pathLit = PatternAstConstant (ContainStr "/")
         ||| PatternAstConstant (ContainStr "\\\\")
+
+-- | 'Inspection' - use 'Text' or 'ByteString'  instead of 'String' @STAN-0216@.
+stan0216 :: Inspection
+stan0216 = Inspection
+    { inspectionId = Id "STAN-0216"
+    , inspectionName = "String"
+    , inspectionDescription = "Usage of the default 'String' Haskell type"
```

Comment:
And for the description, we can give a reason why we suggest this 🙂 (As @jfmengels suggests as well 👍🏼  )